package/src/components/Select/helpers/MultiValue.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import PropTypes from "prop-types";
+import clsx from "clsx";
+import Chip from "../../Chip";
+
+/**
+ * @name MultiValue
+ * @param {Object} props Component props
+ * @returns {React.Component} A React Component
+ */
+export default function MultiValue(props) {
+  return (
+    <Chip
+      size="small"
```

Comment:
The Chip should also use these props: 

```       
variant="default"
color="secondary"
```

Once the Chip PR is merged, and this component has these props assigned, it should look correct: https://deploy-preview-60--heuristic-perlman-0ef024.netlify.com/#/Components/Content/Chip